ajax/libs/weld/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "weld",
+  "version": "0.2.3",
+  "description": "Template antimatter for JavaScript",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tmpvar/weld.git"
+  },
+  "author": "hij1nx <hij1nx@gmail.com>",
+  "license": "MIT",
+  "filename": "weld.min.js",
+  "keywords": [
+    "templates",
+    "templating",
+    "jsdom",
+    "unobtrusive"
+  ],
```

Comment:
@hare1039 I don't think `https://www.weld.io/` is the official homepage for this project. Are you sure?